src-web/components/common/OverviewCards/index.js

```diff
@@ -279,33 +298,71 @@ class OverviewCards extends React.Component {
             }
           ]}
         />
+        {appOverviewCardsData.isArgoApp && (
+          <Card>
+            <CardBody>
+              <AcmActionGroup>
+                <AcmButton
+                  href={this.state.argoLink}
+                  variant={ButtonVariant.link}
+                  id="launch-argocd-editor"
+                  component="a"
+                  rel="noreferrer"
+                  icon={<ExternalLinkAltIcon />}
+                  iconPosition="right"
+                >
```

Comment:
1. Did you consider whether we want the link to open in the same tab or a new one? I think design has opinions on this.
2. Should we grey out or not display this link if for some reason we cannot get the URL for it?

Reply:
Thanks @KevinFCormier , will look into those two scenarios